azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/ListMojo.java

```diff
@@ -19,32 +21,9 @@
 @Mojo(name = "list")
 public class ListMojo extends AbstractFunctionMojo {
 
-    protected static final String TEMPLATES_START = ">> templates begin <<";
-    protected static final String TEMPLATES_END = ">> templates end <<";
-
-    protected static final String BINDINGS_START = ">> bindings begin <<";
-    protected static final String BINDINGS_END = ">> bindings end <<";
-
-    protected static final String RESOURCES_START = ">> resources begin <<";
-    protected static final String RESOURCES_END = ">> resources end <<";
-
-    protected static final String TEMPLATES_FILE = "/templates.json";
-    protected static final String BINDINGS_FILE = "/bindings.json";
-    protected static final String RESOURCES_FILE = "/resources.json";
-
     @Override
     protected void doExecute() throws Exception {
-        info(TEMPLATES_START);
-        printToSystemOut(TEMPLATES_FILE);
-        info(TEMPLATES_END);
-
-        info(BINDINGS_START);
-        printToSystemOut(BINDINGS_FILE);
-        info(BINDINGS_END);
-
-        info(RESOURCES_START);
-        printToSystemOut(RESOURCES_FILE);
-        info(RESOURCES_END);
+    	new ListHandler().execute();
```

Comment:
We could keep `ListMojo` as this is only used for VSCode Function Extension, which will not be used in other toolkits.

Reply:
It is better to be in common, since it is simple without any dependency of maven.